lib/zipkin-tracer/faraday/zipkin-tracer.rb

```diff
@@ -17,6 +17,7 @@ def call(env)
         b3_headers.each do |method, header|
           env[:request_headers][header] = trace_id.send(method).to_s
         end
+        env[:request_headers]['X-B3-Sampled'] = env[:request_headers]['X-B3-Sampled'] == 'true' ? '1' : '0'
```

Comment:
Oh I see, you need this because the TraceId object itself generate true/false.

Then what I'd do is making a method , maybe in the trace module that takes a TraceId object and returns a TraceId object which true/false has been converted to 1/0

Then use that in both rack and faraday because that ensures that if someone else (older versions of ruby zipkin-tracer or other implementations) send you a true/false you understand them but when you sent, you only sent 1/0

Also I'd up the version to 0.19.0 , even 1.0, some day we have to do it. Because it may break compatibility with other implementations (certainly with older versions of the client)